ibis/pyspark/tests/conftest.py

```diff
@@ -15,6 +16,16 @@ def client():
     df = df.withColumn("str_col", F.lit('value'))
     df.createTempView('basic_table')
 
+    df_nans = client._session.createDataFrame(
+        [
+            [np.NaN, 'Alfred', None],
+            [27.0, 'Batman', 'motocycle'],
+            [3.0, None, 'joker'],
```

Comment:
Why is joker a toy? :)

Reply:
(Ignore this comment)

Reply:
You will know if you figure out who is None :)

Reply:
(Ignore this comment)